# All lines beginning with `# DP:' are a description of the patch.
# DP: Description: Fix lang_* fields in LC_ADDRESS
# DP: Related bugs: 
# DP: Dpatch author: Denis Barbier
# DP: Patch author: Denis Barbier
# DP: Upstream status: Not submitted
# DP: Status Details:
# DP: Date: 2006-01-20

---
 localedata/locales/tr_TR  |    6 +++---
 localedata/locales/wal_ET |    1 -
 2 files changed, 3 insertions(+), 4 deletions(-)

Index: langpack-locales/locales/tr_TR
===================================================================
--- langpack-locales.orig/locales/tr_TR	2012-03-06 09:38:51.000000000 +0100
+++ langpack-locales/locales/tr_TR	2012-03-06 10:02:25.024643462 +0100
@@ -3593,10 +3593,10 @@
 country_ab3	"<U0054><U0055><U0052>"
 % Turkish
 lang_name	"<U0054><U0075><U0072><U006B><U0069><U0073><U0068>"
-% tr
+% tur
 lang_term	"<U0074><U0075><U0072>"
+% tur
+lang_lib	"<U0074><U0075><U0072>"
 % tr
-lang_lib	"<U0074><U0072>"
-%tr
 lang_ab		"<U0074><U0072>"
 END LC_ADDRESS
